demisto_sdk/commands/lint/commands_builder.py

```diff
@@ -72,6 +74,52 @@ def build_bandit_command(files: List[Path]) -> str:
     return command
 
 
+def build_xsoar_linter_command(files: List[Path], support_level: str = "base") -> str:
+    """ Build command to execute with xsoar linter module
+    Args:
+        files(List[Path]): files to execute lint
+        support_level: Support level for the file
+
+    Returns:
+       str: xsoar linter command using pylint load plugins
+    """
+    support_levels = {
+        'base': 'base_checker',
+        'community': 'base_checker,community_level_checker',
+        'partner': 'base_checker,community_level_checker,partner_level_checker',
+        'certified partner': 'base_checker,community_level_checker,partner_level_checker,'
+                             'certified_partner_level_checker',
+        'xsoar': 'base_checker,community_level_checker,partner_level_checker,certified_partner_level_checker,'
+                 'xsoar_level_checker'
+    }
+    plugin_path = Path(__file__).parent / 'resources' / 'pylint_plugins'
+    checker_path = ""
+    message_enable = ""
+    if support_levels.get(support_level):
+        checkers = support_levels.get(support_level)
+        support = checkers.split(',') if checkers else []
+        for checker in support:
+            checker_path += f"{plugin_path}/{checker},"
+            checker_msgs_list = Msg_XSOAR_linter.get(checker, {}).keys()
+            for msg in checker_msgs_list:
+                message_enable += f"{msg},"
+
+    command = "python -m pylint"
```

Comment:
I think you need to use the proper python version (similar to what we do with flake8) (2 or 3 depending upon the integration).